### DISCUSSION TITLE
best approach showing a small amount of strings within a cell of q-table

### DISCUSSION THREAD

**Discussion by zanzara** [quasarframework/quasar#8754] [General - Components / Directives / etc] [answered]
I'm using a q-table and one item will be a string array by the backend  and I want to show this strings in a cell.
It would be great if I could use colored, densed q-chip component for that, but i don't know if this is feasable and how to do it.

Otherwise, should I use a textrea type for that and map the string array row somehow with the column formatter function to add at least some CRs for each string?

What would be the best approach.
Thy in advance.

**Comment by metalsadman** [comment, marked as answer]
Use body-cell-[name] slot and customize what you want to display for that cell.

**Reply by zanzara** [reply]
Ok, I see it right now in the examples. Great hint. Thx.
This could be the right way to tinker a bit with different formats/components which could fit best.
